.github/create-release.yml

```diff
@@ -0,0 +1,30 @@
+# https://github.com/actions/create-release
```

Comment:
I do not understand how this workflow is making things any easier. Not saying it's bad, but producing a GitHub release (with a changelog !) now takes literally two clicks and it's not like we're doing it frequently anyway.